### ISSUE TITLE
Showing refinement hierarchy in ProB2UI

### ISSUE THREAD

**Issue by SeeBasTStick** [hhu-stups/prob-issues#224]
It would be nice to visually show the refinement hierarchy between machines in the UI. Maybe with a tree view where the machines in the project are shown?

**Comment by leuschel** [comment]
The Graph Visualisation View has two dot visualisations that show refinement links; but I guess you want something 
![Screenshot 2022-10-10 at 11 11 55](https://user-images.githubusercontent.com/1742246/194833228-12f00885-d3c7-48f0-9f76-9845b7d2825e.png)
![Screenshot 2022-10-10 at 11 12 21](https://user-images.githubusercontent.com/1742246/194833311-03b2f55d-762b-4ce3-a313-e46db9059ee3.png)

Side-note: I have noted that the Event Hierarchy view is quite difficult to scroll in JavaFX.

**Comment by SeeBasTStick** [comment]
Yes exactly, I am aware of this view. For day to day work it would be nice to somehow show it in the main editor window, to navigate the machines more easily. However, I am not sure if this is feasible for very large projects.

![image](https://user-images.githubusercontent.com/49205030/194834975-bc5233fe-5d76-4596-8c36-a0268da08354.png)
